versioned_docs/version-stable/nym-simple-websocket-tutorial.md

```diff
@@ -0,0 +1,658 @@
+---
+sidebar_label: Nym Websocket Tutorial
+description: "Install and use the Nym Wallet."
+hide_title:  false
+title: Nym Simple Websocket Tutorial (Typescript)
+---
+
+### Prerequisites
+
+* Node.js (and Node Package Manager)
+
+#### Installation 
+
+(Windows and macOS)
+Go to https://nodejs.org/en/download/
+
+(Linux - Ubuntu)
+Type and enter the following in your terminal:
+
+```
+sudo apt install curl
+curl -fsSL https://deb.nodesource.com/setup_14.x | sudo -E bash -
+sudo apt-get install -y nodejs
+```
+
+Ensure that after installing Node.js of any of the above operating systems , type and enter the followinginto your terminal to check if the install was successful :
+```
+node --version
+npm --version
+```
+
+
+### Installing Typescript and Bundling our Application
+
+Inside your terminal, type and enter:
+```
+npm install -g typescript
+```
+This will install typescript for us globally. The -g directive tells npm to install the package in the global shared node_modules folder usually at the root folder in your computer
+
+After this , go ahead and create a new folder to start your project in and give it an appropriate name eg; ‘Mixnet Websocket Project’. Continue to then do the following:
+
+1. Open your terminal in the folder you created (or by `cd`'ing to the directory), type and enter:
+
+    ```
+    npm init
+    ```
+    The following chunk of output will then be presented to you. The terminal will prompt your to provide some input for the the sections name to  license. You can simply just press enter after each prompt (like the example below) and it will just set default values. Alternatively, you can go ahead and fill out each value as you wish. 
+
+    <img src="/img/tutorials/simple-websocket/image1.png"/>
```

Comment:
I think we can just include the contents of files in code blocks, instead of using screenshots. Nicer for the formatting